addons/rating/models/rating_mixin.py

```diff
@@ -27,9 +27,25 @@ class RatingMixin(models.AbstractModel):
 
     @api.depends('rating_ids.rating', 'rating_ids.consumed')
     def _compute_rating_last_value(self):
+        # Pure SQL instead of calling read_group to allow ordering array_agg
+        self.flush(['rating_ids'])
+        if not self.ids:
+            self.rating_last_value = 0
+            return
+        self.env.cr.execute("""
+            SELECT
+                array_agg(rating ORDER BY write_date DESC) AS "ratings",
```

Comment:
Note: usage in eLearning could effectively lead to lots de ratings for some res_id (like a course "RCO Dancing on AL laptop" would probably have a lot of ratings). So maybe it could be worth checking it :) .

Reply:
I leave it up to you to decide whether this is good enough.
I simply wanted to point out a _potential_ issue, and make sure you are aware of it :wink: